qiskit/ignis/experiments/calibration/methods/cross_resonance.py

```diff
@@ -0,0 +1,88 @@
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Data source to generate schedule."""
+
+
+from typing import Optional, List
+
+from qiskit.ignis.experiments.calibration import CircuitBasedGenerator
+from qiskit.ignis.experiments.calibration.data_processing import DataProcessingSteps
+from qiskit.ignis.experiments.calibration.cal_base_analysis import BaseCalibrationAnalysis
+from qiskit.ignis.experiments.calibration.instruction_data import InstructionsDefinition
+from qiskit.ignis.experiments.calibration.cal_base_experiment import BaseCalibrationExperiment
+from qiskit.ignis.experiments.calibration.analysis.trigonometric import CosinusoidalFit
+
+
+class RoughCRAmplitude(BaseCalibrationExperiment):
+    """Performs a rough amplitude calibration by scanning the amplitude of the pulse."""
+
+    def __init__(self,
+                 inst_def: InstructionsDefinition,
+                 qc: int,
+                 qt: int,
+                 data_processing: DataProcessingSteps,
+                 amp_vals: List,
+                 gate_name: str,
+                 pulse_names: List[str],
+                 calibration_group: Optional[str] = 'default',
+                 analysis_class: Optional[BaseCalibrationAnalysis] = None,
+                 job: Optional = None):
+        """Create new rabi amplitude experiment.
+        Args:
+            inst_def: The class that defines the instructions for this calibration.
+            qc: Control qubit.
+            qt: Target qubit.
+            data_processing: Steps used to process the data from the Result.
+            amp_vals: Amplitude values to scan in the calibration.
+            analysis_class: Analysis class used.
+            job: Optional job id to retrieve past experiments.
+            gate_name: Name of the gate from the instructions definition.
+            pulse_names: Pulse names in the database entry to provide parameter set to
+                construct pulse schedule to calibrate. By default pi pulse parameter is used.
+        """
+
+        # todo get qubit property from other database.
+        # channel ref frequency is different from pulse sideband and thus
+        # this value is stored in another relational database.
+        # something like qubit property table where f01, anharmonicity, T1, T2, etc... exist.
+        freq01 = None
+
+        u_ch = inst_def.get_channel_name((qc, qt))
+
+        free_names = []
+        for pulse_name in pulse_names:
+            scope_id = inst_def.get_scope_id(gate_name, (qc, qt))
+            free_names.append('%s.%s.%s.amp' % (pulse_name, u_ch, scope_id))
+
+        template_circ = inst_def.get_circuit('xp', (qc, ))
```

Comment:
why we cannot get `cr` entry from the schedule detabase? I think `cr` is a fundamental element and we can assume user have this definition in the database (or at least we can provide helper function to initialize schedule database with `cr`).

Reply:
`gate_name` has now been changed to `cr_name` and accepts `cr` as default.